src/fmu/dataio/_metadata.py

```diff
@@ -305,7 +308,8 @@ def _populate_meta_file(self):
             logger.info("Compute MD5 sum for tmp file...")
             _, self.meta_file["checksum_md5"] = export_file_compute_checksum_md5(
                 self.obj,
-                "tmp",
+                "tmp",  # type: ignore
+                # tmp = true given, this arg is not needed.
```

Comment:
The filename arg might still be needed, maybe the flag should go. Booleans arguments are a reliable code smell for a second function, even if not very DRY

Reply:
yeah.